pagopa/io-web-profile
# [IOPID-1235] : Snackbar Width

## Short description

As described in[ the Jira ](https://pagopa.atlassian.net/browse/IOPID-1235)task, it is requested to ensure that the snackbar is displayed at 100% on mobile.

## List of changes proposed in this pull request

- This PR introduces the custom hook isMobile()

## How to test

Launch the project and navigate to the unlock code creation page. Click on 'copy code' and check on both mobile and desktop how the snackbar is displayed.
